config/config.go

```diff
@@ -320,6 +312,22 @@ func (c *Config) GetStatsFlushInterval() time.Duration {
 	return viper.GetDuration("stats.flush_interval")
 }
 
+// GetCommittedTierFSParams returns parameters for building a tierFS.  Caller must separately
+// build and populate Adaptor.
+func (c *Config) GetCommittedTierFSParams() *pyramidparams.Params {
+	logger := logging.Default().WithField("module", "pyramid")
+	return &pyramidparams.Params{
+		FSName:             "committed",
+		Logger:             logger,
+		Adaptor:            nil, // Built by factory during construction
```

Comment:
When and how is this passed?

Reply:
Sorry, went back-and-forth on where this is generated and ended up with nowhere.